src/pages/apis/drug/remsspl/_infographics.yaml

```diff
@@ -0,0 +1,33 @@
+- title: "REMS SPL Submissions over time"
+  short: "Submissions over time"
+  dataset: REMSSPL
+  description:
+    - "The openFDA REMS SPL API provides data for the drug safety program Risk Evaluation Mitigation Strategies. Since mid-2016, REMS data has been submitted in the [Structured Product Labeling (SPL) format](/data/spl/)."
+    - "This chart shows labeling submissions over time, by looking at their 'effective dates.'"
+  countParam: "effective_time"
+  filters:
+    - title: "All REMS SPL submissions"
+      searchParam: ""
+    - title: "Sublingual Route of Administration"
+      searchParam: "openfda.route:SUBLINGUAL"
+    - title: "Buccal Route of Administration"
+      searchParam: "openfda.route:BUCCAL"
```

Comment:
@violetcrestedwren These two routes produce no data. Might need to replace them with something else.

Reply:
Fixed